src/app/Routes.tsx

```diff
@@ -41,7 +40,7 @@ const ZonesList = lazy(() => import("@/app/zones/views/ZonesList"));
 
 const Routes = (): JSX.Element => (
   <ReactRouterRoutes>
-    <Route element={<Redirect to={urls.machines.index} />} path={urls.index} />
+    <Route element={<Navigate to={urls.machines.index} />} path={urls.index} />
```

Comment:
You might want to add the replace prop to Navigate. see https://reactrouter.com/en/main/upgrading/v5#remove-redirects-inside-switch:~:text=Note%3A%20Be%20aware%20that%20the%20v5%20%3CRedirect%20/%3E%20uses%20replace%20logic%20by%20default%20(you%20may%20change%20it%20via%20push%20prop)%2C%20on%20the%20other%20hand%2C%20the%20v6%20%3CNavigate%20/%3E%20uses%20push%20logic%20by%20default%20and%20you%20may%20change%20it%20via%20replace%20prop.